### PR TITLE
[Nomination] Add ST representative to Security group

### PR DESCRIPTION
**Pull request by yroux**
I'd like to nominate myself to join the LLVM Security group as a representative of ST. I work in ST's compiler team contributing to upstream (LLVM and GNU) and several downstream toolchains. We believe that it is important for us to be part of this group to address or report any potential security issues the LLVM project or our toolchains may encounter.

### CODE REVIEW

**Comment by DimitryAndric** [comment]
As with a previous case of "self-nomination", it would be nice if there was a reference from somebody who is already known in this security-related community, who can vouch for @yroux.

**Review by kbeyls — APPROVED** [review]

**Review by smithp35 — COMMENTED** [review]
LGTM too. I also have known Yvan for many years.

**Review by cuviper — APPROVED** [review]

**Review by andykaylor — APPROVED** [review]

**Review by emaste — APPROVED** [review]
Confirmation from @kbeyls is good enough for me

**Review by pogo59 — APPROVED** [review]
Just curious what "ST" is? LGTM

**Review by DimitryAndric — APPROVED** [review]
ST is STMicroelectronics, who interestingly have the domain `st.com`: https://www.st.com/

**Review by ahmedbougacha — APPROVED** [review]

**Comment by mrragava** [comment]
LGTM

**Comment by kbeyls** [comment]
If I counted correctly, we've got 9 approvals so far.
We currently have 20 people on the security team, so per https://llvm.org/docs/Security.html#choosing-new-members, we'll need another 1 or 2 approvals from existing security group members.

**Review by serge-sans-paille — APPROVED** [review]

**Review by gburgessiv — APPROVED** [review]
Apologies for the latency here - LGTM :)

**Comment by yroux** [comment]
Thanks a lot for your Trust!

I'll merge the PR since the majority is now reached and the 5 business days are passed.

Hope to see you in the next sync meeting

**Pull request merged**
